packages/dev/docs/src/docs.css

```diff
@@ -730,6 +831,24 @@ h2.sectionHeader {
   }
 }
 
+@media (max-width: 375px) {
+  .docSearchBoxThemeProvider {
+    .docSearchBox {
+      max-width: unset;
+      > div,
+      > div > div {
+        min-width: calc(
```

Comment:
i want to revisit these in the future with comments because I have no idea what this or the other calcs in this PR are doing